src/components/date-picker/Datepicker.css

```
@@ -0,0 +1,56 @@
+/* PeakFit DatePicker Global Styles */
+
+.peakfit-datepicker-range,
+.peakfit-datepicker {
+  --rdp-cell-size: 42px;
+  --rdp-accent-color: var(--primary-500);
+  --rdp-background-color: transparent;
+  padding: 0 0.5rem;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)